### PR TITLE
[zh-cn] update 'control mechanisms'

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

Translate accidentally unfinished page

### Related issues and pull requests

part of #14272

### CODE REVIEW

File: files/zh-cn/games/techniques/control_mechanisms/index.md [hunk 1]
```diff
@@ -1,67 +1,69 @@
...
+title: 实现游戏控制机制
 slug: Games/Techniques/Control_mechanisms
+l10n:
+  sourceCommit: 56db19e6b8d19932c1b6150bc42e752e12a2b21f
 ---
 
 {{GamesSidebar}}
 
-One of HTML5's main advantages as a game development platform is the ability to run on various platforms and devices. Streamlining cross device differences creates multiple challenges, not least when providing appropriate controls for different contexts. In this series of articles we will show you how you can approach building a game that can be played using touchscreen smartphones, mouse and keyboard, and also less common mechanisms such as gamepads.
+作为游戏开发平台，HTML5 的主要优势之一是能够在各种平台和设备上运行。简化跨设备差异带来了多重挑战，尤其是在为不同环境提供适当控件时。在本系列文章中，我们将向你展示如何构建一款可以使用触摸屏智能手机、鼠标和键盘以及游戏手柄等不常用机制进行游戏的游戏。
```
Comment (yin1999) [inline files/zh-cn/games/techniques/control_mechanisms/index.md:10]:
```suggestion
作为游戏开发平台，HTML5 的主要优势之一是能够在各种平台和设备上运行。简化跨设备差异带来了多重挑战，尤其是在为不同环境提供适当控件时。在本系列文章中，我们将向你展示如何构建一款可以使用触摸屏智能手机、鼠标和键盘以及游戏手柄等不常用机制进行游玩的游戏。
```

File: files/zh-cn/games/techniques/control_mechanisms/index.md [hunk 2]
```diff
@@ -1,67 +1,69 @@
...
+![Captain Rogers: Battle at Andromeda——游戏项目的文件夹结构，其中包含 JavaScript 源代码、图像和字体。](captainrogers2-folderstructure.png)
 
-As you can see there are folders for images, JavaScript files, fonts and sound effects. The `src` folder contains the JavaScript files split as a separate states — `Boot.js`, `Preloader.js`, `MainMenu.js` and `Game.js` — these are loaded into the index file in this exact order. The first one initializes Phaser, the second preloads all the assets, the third one controls the main menu welcoming the player, and the fourth controls the actual gameplay.
+正如你所看到的，这里有用于存放图片、JavaScript 文件、字体和音效的文件夹。`src` 文件夹包含 JavaScript 文件，这些文件被分为不同的状态：`Boot.js`、`Preloader.js`、`MainMenu.js` 和 `Game.js`。第一个状态初始化 Phaser，第二个状态预加载所有资源，第三个状态控制欢迎玩家的主菜单，第四个状态控制实际游戏。
 
-Every state has its own default methods: `preload()`, `create()`, and `update()`. The first one is needed for preloading required assets, `create()` is executed once the state had started, and `update()` is executed on every frame.
+每个状态都有自己的默认方法：`preload()`、`create()` 和`update()`。第一个方法用于预载所需资源，`create()` 在状态启动后执行，而 `update()` 则在每一帧上执行。
```
Comment (yin1999) [inline files/zh-cn/games/techniques/control_mechanisms/index.md:30]:
```suggestion
每个状态都有自己的默认方法：`preload()`、`create()` 和 `update()`。第一个方法用于预载所需资源，`create()` 在状态启动后执行，而 `update()` 则在每一帧上执行。
```

**Review by yin1999 — APPROVED** [review]
LGTM. Just two nits/improvements

**Comment by 09306677806** [comment]
Better mechanisms should be put into the program that will work like artificial intelligence and a different work sample with better appeal

**Pull request merged**